lib/Importer/TFLiteModelLoader.cpp

```diff
@@ -1968,6 +1971,46 @@ Error TFLiteModelLoader::loadSlice(const tflite::Operator *op,
   return setOutputNodeValue(op, output);
 }
 
+Error TFLiteModelLoader::loadResizeBilinear(const tflite::Operator *op,
+                                            const OperatorInfo &opInfo) {
+  NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getInputNodeValue(op, 0));
+  NodeValue size;
```

Comment:
I think here it would be simpler and less error prone to not use the ```size``` input operand and use directly the output type which can be derived like this:
```
TypeRef outTy;
ASSIGN_VALUE_OR_RETURN_ERR(outTy, getOutputType(op, 0));
```
We always have the output type available because TensorFlowLite is statically typed. We do this similarly for other nodes (e.g. Reshape) so please take a look at the ```loadReshape()``` function.

Reply:
Done,  thanks